src/components/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { NOTIFICATION } from '../../types/Notification';
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+type Props = {
+  notification: NOTIFICATION;
+};
+
+export const Notifications: React.FC<Props> = ({ notification }) => {
+  const [isHidden, setIsHidden] = useState(false);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setIsHidden(true);
+    }, 3000);
```

Comment:
If you want to use timeout to artificially extended closing time of notifications , you should remember to clean it up after usage

Reply:
> If you want to use timeout to artificially extended closing time of notifications , you should remember to clean it up after usage

fixed

Reply:
It is not fixed.
Please refer to this section in React docs to see how we can clean up timeouts in useEffect:
https://react.dev/learn/synchronizing-with-effects#putting-it-all-together